.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,14 +1,16 @@
 ## Purpose:
 What issue(s) does your PR address?
 
-## Results
-If applicable, briefly report your findings.
+## Strategy
+What was your strategy for this new or edited analysis?
+
+## Concerns/Questions for reviewers:
+What things should reviewers look out for?
 
 ## Pull Request Check List:
-* [ ] Run a linter
-* [ ] Set the seed (if applicable)
-* [ ] Comments and/or documentation up to date
-* [ ] Double check your paths
-* [ ] Check headline numbering
-* [ ] Spell check any Rmd file or md file
-* [ ] Restart R and run all notebooks fresh and save
```

Comment:
Is the step to restart R and run all notebooks no longer needed here?

Reply:
Correct. Snakemake does that now.